src/document/DocumentCommandHandlers.js

```diff
@@ -848,6 +859,19 @@ define(function (require, exports, module) {
         return result.promise();
     }
 
+    /**
+     * Checks and handles if auto update is currently in progress
+     */
+    function checkIfAutoUpdateInProgress() {
+        brackets.app.isAutoUpdateInProgress(function (err, isAutoUpdateInProgress) {
```

Comment:
Add `if (brackets.app.isAutoUpdateInProgress) {` to check if this function is registered or not.

Reply:
isAutoUpdateInProgress() function has been added to the core. It is not a registered function. Hence, we need not have this check.

Reply:
I strongly believe suppressing the error would turn out to be harmful in this case, not showing the error to the user would be misleading to him/her believing that the feature can work with just the brackets changes. 
Moreover, I don't see that with any of the other functions' invocations. I believe doing that will be unnecessary bloating of the code, as this is most likely to happen only in the dev machines, which I think should be okay.